stylesheet.css

```
@@ -0,0 +1,34 @@
+body{
+    /*text-align: center;*/
+    color:white;
+        background-image: linear-gradient(
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+}
+form{
+    width: 640px;
+    background-color: rgb(39, 38, 69);
+    color:white;
```

I suggest leaving spaces between semicolons and styling.

---

As mentioned above, kindly be more specific, more explicit and make sure your comment is descriptive and polite! 😄

---

Good job with your styling! Also, remember that to make your code easier to read, you need to leave spaces between semicolons and styling, as follows:
```suggestion
    color:  white;
```